Libplanet/Blockchain/BlockChain.cs

```diff
@@ -438,12 +438,16 @@ internal HashDigest<SHA256> FindBranchPoint(BlockLocator locator)
             {
                 _rwlock.EnterReadLock();
 
+                ImmutableHashSet<HashDigest<SHA256>> indices = Store
+                    .IterateIndex(Id.ToString())
+                    .ToImmutableHashSet();
```

Comment:
It would be more defensive if we make it to chunk these indices to load instead of loading them entirely at a time.  If it's difficult to improve this in that way we'd better leave a `FIXME` comment at least.

Reply:
I've added `FIXME`. (https://github.com/planetarium/libplanet/pull/277/commits/dd197fbca7e766654ebca1210a26205b8b85a325)